dbt/models/iasworld/schema/iasworld.asmt_all.yml

```diff
@@ -57,6 +57,16 @@ sources:
                       AND valclass IS NULL
                   meta:
                     description: at least one class should match pardat class
+              - res_class_matches_pardat:
+                  name: iasworld_asmt_all_ex_matches_pardat_class_ex
+                  major_class_only: false
+                  join_type: inner
+                  additional_select_columns: *select-columns-aggregated
+                  additional_pardat_filter: AND class = 'EX'
+                  config: *unique-conditions
+                  meta:
+                    description: >
+                      Exempt parcels in pardat should be exempt in asmt all
```

Comment:
**[Nitpick]** The output workbook includes the tablename of the failing row as the first column, so I try to avoid repeating the tablename in the test description for the sake of brevity. I also try to start the description with the name of the field that is being tested so that a reader can scan the descriptions and quickly parse out which field failed a test. A better description here (and in the rest of the tests in this PR) would be something like:

```suggestion
                      class should be EX if pardat class is EX
```